pyaerocom/aeroval/coldatatojson_helpers.py

```diff
@@ -116,7 +120,7 @@ def _write_diurnal_week_stationdata_json(ts_data, out_dirs):
     write_json(current, fp, ignore_nan=True)
 
 
-def _add_entry_json(
+def _add_hm_entry_json(
```

Comment:
It this function only used to update `heatmap` JSON files?
If so, it would be better called `_add_heatmap_entry_json`.

If this is a general purpose function, used to update all kids of JSON files,
it would be better to keep the name and rename the `heatmap_file` argument to `file`.

Reply:
This function assumes a certain structure for the json files for writing, which as far as I can tell, is unique to the `hm` directory. It is not called anywhere else, and so changing to `_add_heatmap_entry_json` is probably the most appropriate course of action.

Reply:
When I search for `_add_entry_json` in the codebase, I only find it in `coldatatijson_engine.py` (and the associated test files). I'm happy either way. Honestly I forgot I made this change during testing and could easily live with it just being `_add_json_entry()` . Whatever you think is best

Reply:
I just want good function names. I prefer `_add_heatmap_entry_json` over `_add_entry_json`, but it is up to you.

Reply:
I will stick with `_add_heatmap_entry_json` then